resources/windows/wix/directorylist.wxs.erb

```diff
@@ -0,0 +1,16 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+
+ <!--
+  This is how to include wxi files.  We expect this file to define variables
+  like the version numbers and the upgrade UUID.
+  -->
+  <?include $(sys.SOURCEFILEDIR)include/puppet.wxi ?>
```

Comment:
You don't need this line. It previously was defining variables to be used by wix, but we're supplanting that with ruby variables.